txscript/standard.go

```diff
@@ -44,23 +44,27 @@ type ScriptClass byte
 
 // Classes of script payment known about in the blockchain.
 const (
-	NonStandardTy ScriptClass = iota // None of the recognized forms.
-	PubKeyTy                         // Pay pubkey.
-	PubKeyHashTy                     // Pay pubkey hash.
-	ScriptHashTy                     // Pay to script hash.
-	MultiSigTy                       // Multi signature.
-	NullDataTy                       // Empty data-only (provably prunable).
+	NonStandardTy       ScriptClass = iota // None of the recognized forms.
+	PubKeyTy                               // Pay pubkey.
+	PubKeyHashTy                           // Pay pubkey hash.
+	WitnessPubKeyHashTy                    // Pay witness pubkey hash.
```

Comment:
I think these should contain the script version in them since they only apply to version 0 scripts.

`WitnessV0PubKeyHashTy`
`WitnessV0ScriptHashTy`